src/components/ContentSidebar/withErrorHandling.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @flow
+ * @file withErrorHandling higher order component
+ * @author Box
+ */
+
+import * as React from 'react';
+import ErrorMask from 'box-react-ui/lib/components/error-mask/ErrorMask';
+import InlineError from 'box-react-ui/lib/components/inline-error/InlineError';
+
+import type { MaskError, InlineError as InlineErrorType } from '../../flowTypes';
+
+type Props = {
+    maskError?: MaskError,
+    inlineError?: InlineErrorType
+};
+
+const withErrorHandling = (WrappedComponent: React.ComponentType<any>) => ({
+    maskError,
+    inlineError,
+    ...rest
+}: Props) => {
+    const getWrappedComponent = () => <WrappedComponent {...rest} />;
```

Comment:
Is this a function for some error check?
Or could `<WrappedComponent {...rest} />` be inline below without being a function?